beacon_node/network/src/beacon_processor/mod.rs

```diff
@@ -121,6 +122,14 @@ const MAX_GOSSIP_PROPOSER_SLASHING_QUEUE_LEN: usize = 4_096;
 /// before we start dropping them.
 const MAX_GOSSIP_ATTESTER_SLASHING_QUEUE_LEN: usize = 4_096;
 
+/// The maximum number of queued `SyncCommitteeMessage` objects that will be stored before we start dropping
+/// them.
+const MAX_SYNC_MESSAGE_QUEUE_LEN: usize = 2048;
+
+/// The maximum number of queued `SignedContributionAndProof` objects that will be stored before we
+/// start dropping them.
+const MAX_SYNC_CONTRIBUTION_QUEUE_LEN: usize = 1024;
```

Comment:
Max valid messages per slot would be target_aggregators * subnet count = 64. Kept this conservatively high. 
This queue size is 0 for `altair-devnet-1`

Reply:
That sounds good to me too :+1: